docs/TroubleshootingTips.md

```diff
@@ -0,0 +1,63 @@
+---
+id: TroubleshootingTips
+title: Troubleshooting Tips
+---
+
+## Troubleshooting help
+
+If you run across some errors while using Enklu here are a few simple troubleshooting tips you can try. 
+
+First it's recommended to work with [Google Chrome](https://www.google.com/chrome/) or a variant like [Sidekick](https://www.meetsidekick.com/). 
+
+
+
+### Log-in help
+
+To prevent login issues use password managers like [Lastpass](https://www.lastpass.com/) or [Bitwarden](https://bitwarden.com/ ). 
+
+If you still run into a login issue, reach out to us by [email](contact@enklu.com) or check out our [contact page](https://docs.enklu.com/contact).
+
+
+### General tips
+
+If you are running into issues with your scene here are few common solutions.
+
+#### Check for code issues
+
+Check your console by going to your Assets window then click on the Log window tab.
```

Comment:
for readability, maybe change this to: "To check your console, go to the bottom window and click on the Log tab"